.github/workflows/decnum.yml

```diff
@@ -0,0 +1,32 @@
+name: decnum
```

Comment:
Maybe call it no-decnum etc for clarity?

Reply:
I'm okay with this name, this workflow is about the decnum option and there's only a job which disables it.